wrappers/azurekeyvault/azurekeyvault.go

```diff
@@ -253,6 +262,26 @@ func (v *Wrapper) Decrypt(ctx context.Context, in *wrapping.EncryptedBlobInfo, a
 	return wrapping.NewEnvelope(nil).Decrypt(envInfo, aad)
 }
 
+func (v *Wrapper) ImportKey(_ context.Context, _ string, _ keysutil.KeyType, _ keysutil.KeyEntry, _ wrapping.Purpose, _ wrapping.ProtectionLevel) (string, error) {
+	return "", nil
+}
+
+func (v *Wrapper) RotateKey(_ context.Context, _ string, _ keysutil.KeyType, _ keysutil.KeyEntry, _ wrapping.Purpose, _ wrapping.ProtectionLevel) (string, error) {
+	return "", nil
+}
+
+func (v *Wrapper) DeleteKey(_ context.Context, _ string) (bool, error) {
+	return false, nil
+}
+
+func (v *Wrapper) EnableKeyVersion(_ context.Context, _, _ string) error {
+	return nil
+}
+
+func (v *Wrapper) DisableKeyVersion(_ context.Context, _, _ string) error {
+	return nil
+}
+
```

Comment:
Are these intentionally not implemented?

Reply:
Yep, it's intentional. I wanted to keep this PR focused on the interface. The implementations will come next.